### ISSUE TITLE
HDMI overscan issue

### ISSUE THREAD

**Issue by IvanPizhenko** [loboris/OrangePi-BuildLinux#4]
Hi, I'm facing HDMI overscan issue with opi+. Screen image goes out of screen borders. Is there something within orange pi like /boot/config.txt for Raspberry Pi, which enables to control overscan parameters? (see description here http://www.opentechguides.com/how-to/article/raspberry-pi/28/raspi-display-setting.html) Or is there any other way to fix the issue?

**Comment by chrnybo** [comment]
ivanp2015,

Have you found any other way of controlling overscan?

**Comment by IvanPizhenko** [comment]
No, unfortunately not found yet. Looks like nobody knows. So i just have given up, and living somehow with overscan. Looks like Orange Pi is poorely supported, or if not so, I miss the point. But actually, what I've learned is that it is better to pay extra $10 for original Raspberry Pi and don't have at all or have much less of all this headache....

**Comment by ivanp2015** [comment]
Orange Pi is poorly supported, or miss the point. What I've learned it is better to pay extra $10 for original Raspberry Pi and don't have or much less of all this headache....
